config/hardcoded_config.go

```diff
@@ -352,6 +325,18 @@ func (c *config) GossipListenPort() uint16 {
 	return uint16(c.kv[GOSSIP_LISTEN_PORT].Uint32Value)
 }
 
+func (c *config) GossipPeers() topologyProviderAdapter.GossipPeers {
```

Comment:
We want to move these from the config and have the config eventually only hold very internal configurations. all the rest will come from url-file in private and Ethereum in public. in internal tests from "in memory" (not config or not directly) and the file-file may be useful in staging env - but it may be removed completely in the future)